plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreConfig.java

```diff
@@ -326,4 +333,30 @@ public ThriftMetastoreConfig setWriteStatisticsThreads(int writeStatisticsThread
         this.writeStatisticsThreads = writeStatisticsThreads;
         return this;
     }
+
+    @Config("hive.metastore.client.http.bearer-token")
+    @ConfigDescription("Http Bearer token to authenticate with a HTTP transport based metastore service")
+    public ThriftMetastoreConfig setHttpToken(String httpToken)
+    {
+        this.httpToken = httpToken;
+        return this;
+    }
+
+    public String getHttpToken()
+    {
+        return httpToken;
+    }
+
+    @Config("hive.metastore.client.http.additional-headers")
```

Comment:
I stumbled by chance on another config property which deals with HTTP headers.

It uses `:` to delimit keys and values instead of `=` 

https://github.com/trinodb/trino/blob/f5eddd80d34b2a90daf169273f083d081acbf603/plugin/trino-http-event-listener/src/main/java/io/trino/plugin/httpquery/HttpEventListenerConfig.java#L97-L116

Reply:
Ah, interesting. I was not aware of this. I can update my PR to use `:` as delimiter to keep it consistent.